README.md

```diff
@@ -21,6 +21,7 @@ All files have a maximum lifetime and are then deleted.
 - Client side caching by HTTP headers `Last-Modified` and `If-Modified-Since` and HTTP status code 304
 - On Linux there is additional hardening through Landlock and seccomp-bpf
 - On OpenBSD there is additional hardening through pledge and unveil
+- Print less output with `?onlyURL` flag, e.g. for use with Weechat-Android.
```

Comment:
```suggestion
- Print only the final UR with the `?onlyURL` GET parameter instead of a more verbose output
```

In this context, it's a parameter not a flag. Furthermore, I don't think it's necessary to advertise WeeChat twice :)